lib/matplotlib/widgets.py

```diff
@@ -4199,19 +4199,26 @@ class Lasso(AxesWidget):
         Whether to use blitting for faster drawing (if supported by the
         backend). See the tutorial :ref:`blitting`
         for details.
+    props: dict, optional
+        Lasso line properties. See `matplotlib.lines.Line2D` for valid
+        properties. Default props are:
+            `{'linestyle' : '-', 'color' : 'black', 'lw' : 2}`.
```

Comment:
```suggestion
        properties. Default props are::

            `{'linestyle' : '-', 'color' : 'black', 'lw' : 2}`.
```

Reply:
This should format it properly and avoid the errors.

An option is to skip the indentation and extra line and to use double backtick which will set it inline in code style.

Reply:
Thanks for the suggestion! Added